README.md

```diff
@@ -117,27 +117,16 @@ These tasks are preconfigured
 
 To test changes from packages in this repository in other applications follow these steps:
 
-1. Run `npm install` in the root of the `frontend-components` working copy
-2. Remove `react` and `react-dom` from `node_modules`
-  ```
-  rm -rf node_modules/react; rm -rf node_modules/react-dom
-  ```
-  This is because we want to use hooks and different reacts are not playing nicely with hooks [facebook/react/issues/15315](https://github.com/facebook/react/issues/15315)
-
-3. Link `react` and `react-dom` from your application. Running from folder that contains your application and frontend components. Running `ls` in this folder would yield `<application-folder>  insights-proxy  frontend-components`
-```
-ln -s $PWD/<application-folder>/node_modules/react frontend-components/node_modules/react
-ln -s $PWD/<application-folder>/node_modules/react-dom frontend-components/node_modules/react-dom
-```
-
-4. Change into the directory of the package you are working on, for example `cd packages/components` and run `npm link`*
-5. Change into the directory of the application you'd like to include the package and run `npm link @redhat-cloud-services/frontend-components`*
+1. Install [yalc](https://github.com/whitecolor/yalc) globally. e.g. `npm install -g yalc`.
```

Comment:
Can you make this as seperate section? I like this idea, let's just allow consumers to choose which one they prefer.

Reply:
Sure, I'll put into a separate section.